.github/scripts/mr_generate_summary.py

```diff
@@ -0,0 +1,61 @@
+# Collect the results of the various model test runs which are done as part of
+# the model regression CI pipeline and dump them as a single file artifact.
+# This artifact will the then be published at the end of the tests.
+import json
+import os
+
+summary_file = os.environ["SUMMARY_FILE"]
+config = os.environ["CONFIG"]
+dataset = os.environ["DATASET_NAME"]
+task_mapping = {
+    "intent_report.json": "intent_classification",
+    "CRFEntityExtractor_report.json": "entity_prediction",
+    "DIETClassifier_report.json": "entity_prediction",
+    "response_selection_report.json": "response_selection",
+}
+data = {}
+
+
+def generate_json(file, task):
+    global data
```

Comment:
why do we need a `global` here? Can't we just return the value?

Reply:
Good point, it doesn't have to be a global variable.